### ISSUE TITLE
Model training

### ISSUE THREAD

**Issue by milican04** [milican04/CT-Scans-Segmentation#1]
Model with gradient descending optimization method should be trained on training data.
**Tasks:**
1. edit convert_ct_dataset_to_slices function
2. split training data into train and validation data
3. check u-net architecture one more time. Should we apply dropout after activation or before?
4. train model in approximately 20 epochs
5. evaluate model on validation data
